ngrok-deployment.yml

```
@@ -0,0 +1,40 @@
+apiVersion: v1
+items:
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: ngrok
+  spec:
+    selector:
+      matchLabels:
+        app: ngrok
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          app: ngrok
+      spec:
+        containers:
+        - name: ngrok
+          image: ramene/ngrok
+          command: ["ngrok"]
+          args: ["http", "-hostname=app.example.com", "deliverybot-service"]
```

```suggestion
          args: ["http", "-hostname=app.example.com", "deliverybot-service"]
```
Remeber to update hostname to reflect your own reserved domain